src/storhaug

```diff
@@ -399,6 +402,7 @@ setup_create_resources()
             config="${HA_NFS_CONF}" \
             state_fs="${HA_NFS_TYPE}" \
             state_mnt="${HA_MNT_DIR}/${HA_NFS_MNT_DIR}" \
+        op monitor on-fail="block" \
```

Comment:
Is this a particular detail for NFS-Ganesha? Most resources can safely be restarted (or attempted to be restarted) a few times before blocking them.